cmd/lncli/commands.go

```diff
@@ -695,6 +699,11 @@ var closeChannelCommand = cli.Command{
 				"sat/byte that should be used when crafting " +
 				"the transaction",
 		},
+		cli.StringFlag{
+			Name: "delivery_addr",
+			Usage: "(optional) a BASE58 encoded address to deliver " +
```

Comment:
shouldn't need to be base58, bech32 is also supported.

Reply:
True. Removed the reference to BASE58 altogether.